js/mixins/responsiveManagerMixin.js

```diff
@@ -27,23 +28,47 @@ var ResponsiveManagerMixin = {
     this.generateResponsiveData();
   },
 
+  /**
+   * It gives the value of height, based on the width of mainVideoContainer (in a ratio 16:9);
+   * @param {number} componentWidth - width of the element
+   * @returns {number} - default height of the element
+   * If we do not have width of mainVideoContainer it returns 0
+   */
+  getDefaultElementHeight: function(componentWidth) {
+    const ratioCoef = 0.5625; // y - coefficient for default aspect ratio 16:9
+    let componentHeight = 0;
+    if (componentWidth && Utils.ensureNumber(componentWidth)) {
+      componentHeight = componentWidth * ratioCoef;
+    }
+    return componentHeight;
+  },
+
   generateResponsiveData: function() {
-    var componentWidth = 0;
-    var componentHeight = 1;
-    var dom = ReactDOM.findDOMNode(this);
+    let componentWidth = 0;
+    let componentHeight = 0;
+    const dom = ReactDOM.findDOMNode(this);
```

Comment:
ES6 destructuring would be good here:
const { width, height } = dom.getBoundingClientRect();